Added gaming layout and read me (#7996)
qmk/qmk_firmware: keyboards/maartenwut/wonderland/keymaps/gaming/readme.md

## Gaming Layout

### This layout was designed for everyday work with all necessary keys reachable from the home row, while having a toggle layer moving the space and backspace to a more traditional location. 

> * The main layer gives you access to home, end , pgup and pgdn. While keeping control, delete, backspace, and enter reachable when your fingers are in the home row. 

Reviewer: ```suggestion
* The main layer gives you access to home, end, pgup and pgdn. While keeping control, delete, backspace, and enter reachable when your fingers are in the home row. 
```

* The main layer also includes capslock key while tapped and it becomes control when pressed. 
* **Toggle(1)[Game]:** Toggles into a gaming layout that gives you space on your left hand thumb and moves the backspace to the traditional location. That way you can game one handed while using a mouse.  

> * **MO(2)[_FUNC]:** Is a momentary layer for all special functions including F keys , volume control, and print screen. It also keys character C and V, allowing you to copy and paste while using arrow keys. This is an easy to reach location for all special funtionaliy that is not normally used. 

Reviewer: ```suggestion
* **MO(2)[_FUNC]:** Is a momentary layer for all special functions including F keys , volume control, and print screen. It also keys character C and V, allowing you to copy and paste while using arrow keys. This is an easy to reach location for all special funtionality that is not normally used. 
```

> * **M(3)[_RGB]:** Is a monentary layer that gives you all the RGB control and keyboard Reset. This moves the functionality not normally used to a place you won't accidently hit and get annoyed at.

Reviewer: ```suggestion
* **M(3)[_RGB]:** Is a momentary layer that gives you all the RGB control and keyboard Reset. This moves the functionality not normally used to a place you won't accidentally hit and get annoyed at.
```
